CRM/ACL/BAO/ACL.php

```diff
@@ -530,22 +530,37 @@ private static function loadDenyIDs(int $contactID, string $tableName, int $type
     $aclKeys = array_keys($acls);
     $aclKeys = implode(',', $aclKeys);
     $query = "
-SELECT   a.operation, a.object_id
+SELECT   a.operation, a.object_id, a.weight
```

Comment:
I'm a little concerned about this query causing a crash in an upgrade situation when the column hasn't been added yet. `DAO::getSupportedFields()` is your friend.